### DISCUSSION TITLE
Player view

### DISCUSSION THREAD

**Discussion by sega-gremlen** [gilbarbara/react-spotify-web-playback#173] [Q&A] [answered]
Could you please tell me how to achieve the exact player view shown in your repository's README?

![image](https://github.com/user-attachments/assets/4e01837d-f14b-4514-99c7-8b8c88e19eed)

I am currently getting this type of layout instead:

![image](https://github.com/user-attachments/assets/037e7156-0121-43b3-8a74-e16a5026b7e9)

Thank you!

**Comment by gilbarbara** [comment, marked as answer]
Hey @sega-gremlen 
I forgot to update the image. It's outdated.
You can't change the slider, but there's a prop to show the volume as a button.
